src/main/java/Aly.java

```diff
@@ -0,0 +1,242 @@
+import java.util.Scanner;
+
+public class Aly {
+
+    private static final int MAX_TASKS = 100;
+    private static final String LINE_SEPARATOR = "-----------------------------------------------------------------------------";
+    private static final String LOGO = "    _      _     _   _\n"
+            + "   / \\    | |   \\ \\ / /\n"
+            + "  / _ \\   | |    \\ V / \n"
+            + " / ___ \\  | |__   | |  \n"
+            + "/_/   \\_\\ |____|  |_|  \n";
+    private static Scanner in = new Scanner(System.in);
+    private static Task[] tasks = new Task[MAX_TASKS];
+
+    private static void printLine() {
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    private static void initialise() {
+        boolean isExit = false;
+        while (!isExit) {
+            printLine();
+            System.out.println("I'm hungry, what do you want?");
+            System.out.println("Faster lah! Enter 1 for echo function, 2 for task creation function, 3 for task marking function, 0 to exit");
+            printLine();
+            String input = in.nextLine().trim();
+            switch (input) {
+            case "0":
+                isExit = true;
+                break;
+            case "1":
+                System.out.println("Enter 0 to return to main menu anytime!");
+                printLine();
+                echo();
+                break;
+            case "2":
+                System.out.println("Enter 0 to return to main menu anytime!");
+                printLine();
+                list(tasks);
+                break;
+            case "3":
+                System.out.println("Enter 0 to return to main menu anytime!");
+                printLine();
+                markTask(tasks);
+                break;
+            default:
+                System.out.println("Can read instructions anot? Try again!");
+                break;
+            }
+        }
+        exit();
+    }
+
+    private static void exit() {
+        System.out.println("Bye! I'm going to eat MacDonalds now!");
+        printLine();
+        System.exit(0);
+    }
+
+    private static void echo() {
+        boolean isExit = false;
+        while (!isExit) {
+            System.out.println("Waiting for Input...");
+            String line = in.nextLine().trim();
+            if (line.equals("0")) {
+                isExit = true;
+                System.out.println("Returning to main menu!");
+            } else {
+                printLine();
+                System.out.println(line);
+                printLine();
+            }
+        }
+    }
+
+    private static void list(Task[] listItems) {
+        boolean isExit = false;
+        int index = 0;
+        index = getIndex(listItems, index);
+        while (!isExit) {
+            System.out.println("Enter 'list' to see your list of tasks, 'todo'/'deadline'/'event' to add that respective type of task, '0' to exit");
```

Comment:
The indentation for wrapped lines should be 8 spaces more than the parent line. The split lines are not indented here.